### PR TITLE
Added status as archived

### PR DESCRIPTION
**Pull request by priyanka-28**
Checked for all existing archived repositories in pr-data.csv and marked their status as 'RepoArchived'.

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -4044,8 +4044,8 @@ https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30a
...
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase8,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase9,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
-https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToJSON,ID,Opened,https://github.com/twilio/authy-java/pull/70,
-https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToXML,ID,,,
+https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToJSON,ID,RepoArchived,https://github.com/twilio/authy-java/pull/70,
```
Comment (darko-marinov) [inline pr-data.csv:4047]: Please do not change `Opened` to `RepoArchived` (several times).

File: pr-data.csv [hunk 2]
```diff
@@ -4086,7 +4086,7 @@ https://github.com/visallo/vertexium,f1fd59e23610312cf0ec6756ea01baaf56cdb74e,ac
 https://github.com/visallo/vertexium,f1fd59e23610312cf0ec6756ea01baaf56cdb74e,inmemory,org.vertexium.inmemory.InMemoryGraphTest.manyVisibilities,NOD,,,
 https://github.com/visallo/vertexium,f1fd59e23610312cf0ec6756ea01baaf56cdb74e,inmemory,org.vertexium.inmemory.InMemoryGraphTest.testMarkPropertyVisible,NOD,,,
 https://github.com/visallo/vertexium,bc52f7cd33f6e559b3f144db77e160bddac0e003,core,org.vertexium.util.IncreasingTimeTest.currentTimeMillisReturnsTimeGreaterThanSystemTime,NOD,,,
-https://github.com/vmware/admiral,e4b02936cc7d4ff2714e7231db0c4373ba5d48a2,adapter/kubernetes,com.vmware.admiral.adapter.kubernetes.KubernetesMaintenanceTest.testAdapterRequestOnPeriodicMaintenance,NIO,Opened,https://github.com/vmware/admiral/pull/320,RepoArchived
+https://github.com/vmware/admiral,e4b02936cc7d4ff2714e7231db0c4373ba5d48a2,adapter/kubernetes,com.vmware.admiral.adapter.kubernetes.KubernetesMainteceTest.testAdapterRequestOnPeriodicMaintece,NIO,RepoArchived,https://github.com/vmware/admiral/pull/320,RepoArchived
```
Comment (darko-marinov) [inline pr-data.csv:4089]: Test name was changed here, but the line should not be changed anyhow (do not change `Opened` to `RepoArchived`).

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Thanks for the updates. Please do not change `Opened`. Revise the PR so I can accept it.

File: pr-data.csv [hunk 3]
```diff
@@ -4044,8 +4044,8 @@ https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30a
...
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase8,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase9,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
-https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToJSON,ID,Opened,https://github.com/twilio/authy-java/pull/70,
-https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToXML,ID,,,
+https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToJSON,ID,Opened,https://github.com/twilio/authy-java/pull/70,RepoArchived
```
Comment (darko-marinov) [inline pr-data.csv:4047]: Thanks for the update. If prior changes added `RepoArchived` as a comment to `Opened`, then you should do it for all other cases where you initially replaced `Opened` with `RepoArchived`.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Adding `RepoArchived` as a comment to `Opened` (instead of replacing `Opened` with `RepoArchived`) looks good. Please do it for all the cases.

File: pr-data.csv (hunk 3 above)
Comment (priyanka-28) [reply pr-data.csv:4047]: The previous cases where I replaced Opened with RepoArchived already contain RepoArchived as a comment in pr-data.csv. So, I believe no changes are required for those cases.

Comment (darko-marinov) [reply pr-data.csv:4047]: Ah, okay, I didn't notice that. Did you check all other repos and found none that became `RepoArchived`? You can reply to me by email (and CC Yuntong).

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Please follow up by email.

**Review by xieyt2000 — COMMENTED** [review]
(1) if there's already a status, put 
in notes `RepoArchived` or `RepoDeleted`, and (2) if there's no status 
(i.e., the test was just found but not `Opened`, `Accepted`, or `Rejected`) 
put in the status itself `RepoArchived` or `RepoDeleted`.

Seems that (2) is not implemented.

**Review by xieyt2000 — COMMENTED** [review]
The python code LGTM

**Pull request merged**